de-lib/src/main/java/org/iplantc/de/collaborators/client/views/GroupViewImpl.java

```diff
@@ -122,10 +127,11 @@ public void onFailure(Throwable caught) {}
             @Override
             public void onSuccess(GroupDetailsDialog result) {
                 result.show();
```

Comment:
Good question.  It looks like it's mixed right now whether they're launched from the view or the presenter, but also looks like the majority are from the presenter.  I can refactor for consistency.